### PR TITLE
Feature1

### PR DESCRIPTION
**Pull request by AnelisaDee**
**Create a Hello Microverse project.**
This is my first project with microverse where I created 3 files; index.html, style.css, test.md and the last one needs to be ignored. The project is completed according to these requirements:
 
**Project Requirements**

- A .html file that includes a header with the text "Hello Microverse!".
  -I use h1 tag for this header and created a class inside it.
- A .css file that includes one class with styles for your header.
- Style the header with with the color red.
- Add 2 space indentation inside my .css file.
- A test.md file with the text "This file should be ignored by git".

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+Give a ⭐️ if you like this project!
+
+Acknowledgments
+Hat tip to anyone whose code was used
+Inspiration
+etc
+📝 License
+This project is MIT licensed.
+
+NOTE: we recommend using the MIT license - you can set it up quickly by using templates available on GitHub. You can also use any other license if you wish.
```
Comment (levy002) [inline README.md]: - [ ] KIndly make sure to clone the Microverse [README template](https://github.com/microverseinc/readme-template/blob/master/README.md) in your repo in order to make the repo looks more professional and try to customize it according to your project. 

   **_Make sure your README looks more professional by defining the headers of each section. Kindly follow the provided template to achieve it_**

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @AnelisaDee   , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Good job for using the correct github-flow :heavy_check_mark: 
- All linters are passing :heavy_check_mark: 
- Good PR description :heavy_check_mark: 

## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by AnelisaDee** [comment]
Thank you for the positive feedback. I will make the required changes :)

**Review by victorokoroji — APPROVED** [review]
Hi @AnelisaDee, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The README is well documented ✅ 
- Linters are working properly ✅ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**